src/components/editor-page/editor-pane/one-dark.scss

```diff
@@ -1,11 +1,7 @@
-/*
- * Copyright (c) 2015 Hikio - twitter.com/hik_io
- * SPDX-FileCopyrightText: 2015 Hikio - twitter.com/hik_io
- *
- * SPDX-License-Identifier: MIT
+/*!
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
  *
- * Atom One Dark Theme
- * GitHub https://github.com/hikio/brackets-one-dark
+ * SPDX-License-Identifier: AGPL-3.0-only
  */
 
 /*
```

Comment:
Same thing as usual for this PR series...

Reply:
Rebased